src/shared/actions.rs

```diff
@@ -1,16 +1,11 @@
 use makepad_widgets::{ActionDefaultRef, DefaultNone};
-use moly_mofa::MofaAgent;
 use moly_protocol::data::FileID;
 
-#[derive(Clone, Debug)]
-pub enum ChatHandler {
```

Comment:
This type is equivalent to `ChatEntityId` (ex-`ChatEntity`), so I removed it and replaced its usage.